test/utilities.jl

```diff
@@ -522,6 +522,16 @@ end
         @test_throws ArgumentError Documenter.Utilities.check_strict_kw(:a)
         @test_throws ArgumentError Documenter.Utilities.check_strict_kw([:a, :doctest])
     end
+
+    @testset "@docerror" begin
+        doc = (; internal = (; errors = Symbol[]), user = (; strict = [:doctest, :setup_block]))
+        foo = 123
+        @test_logs (:warn, "meta_block issue 123") (Documenter.Utilities.@docerror(doc, :meta_block, "meta_block issue $foo"))
+        @test :meta_block ∈ doc.internal.errors
+        @test_logs (:error, "doctest issue 123") (Documenter.Utilities.@docerror(doc, :doctest, "doctest issue $foo"))
+        @test :doctest ∈ doc.internal.errors
+        @test_throws ArgumentError @macroexpand Documenter.Utilities.@docerror(doc, :foo, "invalid tag")
```

Comment:
This throws `LoadError` on some older julia versions, so you have to do
```suggestion
        try
            @macroexpand Documenter.Utilities.@docerror(doc, :foo, "invalid tag")
            error("unexpected")
        catch err
            err isa LoadError && (err = err.error)
            @test err isa ArgumentError
            @test err.msg == "tag :foo is not a valid Documenter error"
        end
```

Reply:
thanks for figuring out how to fix it !